src/ui/public/timefilter/timefilter.js

```diff
@@ -23,6 +23,12 @@ uiModules
   var Events = Private(EventsProvider);
   var diff = Private(UtilsDiffTimePickerValsProvider);
 
+  config.$bind($rootScope, 'dateFormat:dow', 'dateFormat_dow');
+
+  $rootScope.$watch('dateFormat_dow', function (day) {
```

Comment:
I think this needs to move somewhere else, though I'm not sure off-hand where.  In general, I think interacting with scope at all inside of a service is an anti-pattern, but at least the other watchers in this file are limited to the explicit action of instantiating a Timefilter object rather than the act of injecting the timefilter service.  Our services really shouldn't have side effects when they're included.

Is there a controller that this can be moved to, or at the very least can this be moved to where the other watchers in this file are?